docs/PacketFence_Installation_Guide.asciidoc

```diff
@@ -1037,9 +1037,12 @@ Where :
 * *Workgroup* is the workgroup of your domain in the old syntax (like NT4).
 * *DNS name of the domain* is the FQDN of your domain. The one that suffixes your account names.
 * *This server's name* is the name that the server's account will have in your Active Directory.
+* *Sticky DC* is the prefered domain controller to connect to.
```

Comment:
preferred

Reply:
Also, please adjust this list to be a numbered list. Ensure you follow proper standards while doing so